src/workflows/bidsWholeBrainFuncMask.m

```diff
@@ -0,0 +1,37 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function mask = bidsWholeBrainFuncMask(opt)
```

Comment:
the way this runs now, this will only return the filename of the mask of the last subject.

maybe we should make sure it returns a list of masks

Reply:
yup, now it is added. 
if you prefer another way, let me know.